### PR TITLE
SDP unlock!

### PR DESCRIPTION
**Pull request by emreyamangil**
This PR unlocks SDP with convergence bug!

### CODE REVIEW

**Comment by emreyamangil** [comment]
P=NP, Mosek is running noicely!

**Comment by emreyamangil** [comment]
@chriscoey this fixes expDesign!! Just specify CPLEX as MILP solver instead of Cbc, it was indeed sparse matrix causing the issue!

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ julia> Pkg.add("Pajarito")
 
 ## Should I use Pajarito?
 
-Pajarito is a solver for mixed-integer convex optimization problems. These are optimization problems where some variables are restricted to take discrete (e.g. binary) values, and the problem becomes convex when the discreteness constraints are relaxed. If your problem falls into this class, you may consider using Pajarito. Pajarito does not yet support constraints enforcing positive semidefiniteness of a matrix (i.e., mixed-integer semidefinite programming).
+Pajarito is a solver for mixed-integer convex optimization problems. These are optimization problems where some variables are restricted to take discrete (e.g. binary) values, and the problem becomes convex when the discreteness constraints are relaxed. If your problem falls into this class, you may consider using Pajarito. You may specify a mixed-integer semidefinite program through Convex.jl, though this is an experimental feature with the current version of Pajarito (you can turn it on using `misdp=true` at your own risk!).
```
Comment (mlubin) [inline README.md]: Let's not document the sdp option for now

File: src/conic.jl [hunk 2]
```diff
@@ -50,6 +50,7 @@ type PajaritoConicModel <: MathProgBase.AbstractConicModel
     profile::Bool               # Performance profile switch
     disaggregate_soc::Symbol    # Disaggregate SOC constraints following Vielma et al.
     instance::AbstractString    # Path to instance
+    misdp::Bool                 # Indicator for misdp problem
```
Comment (mlubin) [inline src/conic.jl]: How about the name ``enable_sdp``?

File: src/conic.jl [hunk 3]
```diff
@@ -879,7 +912,7 @@ function MathProgBase.optimize!(m::PajaritoConicModel)
 
     # Release the inf_conic_model if applicable
     if applicable(MathProgBase.freemodel!,ini_conic_model)
-        MathProgBase.freemodel!(ini_conic_model)
+        !contains(string(typeof(m.cont_solver)), "MosekSolver") && MathProgBase.freemodel!(ini_conic_model)
```
Comment (mlubin) [inline src/conic.jl]: This is already fixed in mosek (https://github.com/JuliaOpt/Mosek.jl/pull/71), we don't need a special case.

File: src/conic.jl [hunk 4]
```diff
@@ -938,7 +981,7 @@ function MathProgBase.optimize!(m::PajaritoConicModel)
 
             # Release the conic_model if applicable
             if applicable(MathProgBase.freemodel!,conic_model)
-                MathProgBase.freemodel!(conic_model)
+                !contains(string(typeof(m.cont_solver)), "MosekSolver") && MathProgBase.freemodel!(conic_model)
```
Comment (mlubin) [inline src/conic.jl]: Same here

File: test/runtests.jl [hunk 5]
```diff
@@ -15,21 +15,41 @@ import Convex
...
 
+# TODO remove the following 
+myip_solvers = Any[]
+for i = 1:length(ip_solvers)
+    contains(string(typeof(ip_solvers[i])), "MosekSolver") && continue 
+    push!(myip_solvers, ip_solvers[i])
+end
+mylazy_solvers = Any[]
+for i = 1:length(lazy_solvers)
+    contains(string(typeof(lazy_solvers[i])), "MosekSolver") && continue
```
Comment (mlubin) [inline test/runtests.jl]: Mosek isn't listed in ``lazy_solvers``

File: src/conic.jl (hunk 3 above)
Comment (emreyamangil) [inline src/conic.jl]: Hmmm I am still getting freemodel crash from Mosek (I have your fix under git log of Mosek)

Comment (mlubin) [inline src/conic.jl]: What's the crash?

Comment (emreyamangil) [inline src/conic.jl]: convert has no method matching convert(::Type{Mosek.MSTask}, ::Ptr{Void})

**Pull request merged**